### PR TITLE
Skip dim updates if container_id is empty

### PR DESCRIPTION
**Pull request by asuresh4**

### CODE REVIEW

File: internal/monitors/kubernetes/cluster/monitor_test.go [hunk 1]
```diff
@@ -183,6 +192,17 @@ var _ = Describe("Kubernetes plugin", func() {
 			MergeIntoExisting: true,
 		}))
 
+		Expect(dims).ShouldNot(ConsistOf(&types.Dimension{
```
Comment (keitwb) [inline internal/monitors/kubernetes/cluster/monitor_test.go]: What is this test for?

Comment (keitwb) [reply internal/monitors/kubernetes/cluster/monitor_test.go]: Also it is redundant to the previous assertion.

Comment (keitwb) [reply internal/monitors/kubernetes/cluster/monitor_test.go]: ConsistOf will test the length as well as the elements matching.

Comment (asuresh4) [reply internal/monitors/kubernetes/cluster/monitor_test.go]: Oh ok. I wasn't sure if we also needed to assert that a particular Dimension should not exist in the output.

Comment (asuresh4) [reply internal/monitors/kubernetes/cluster/monitor_test.go]: Removed the additional check

**Pull request merged**